src/clib/lib/res_util/subst_list.cpp

```diff
@@ -357,7 +357,8 @@ subst_list_update_string(const subst_list_type *subst_list, char **string) {
    operation has been performed.
 */
 char *subst_list_alloc_filtered_string(const subst_list_type *subst_list,
-                                       const char *string) {
```

Comment:
Need to document what the context string is supposed to do here.

Reply:
it is done.